features/attached_commands.feature

```diff
@@ -584,3 +584,44 @@ Feature: Command behaviour when attached to an UA subscription
            | xenial  |
            | bionic  |
            | focal   |
+
+
+    @wip
+    @series.all
+    @uses.config.machine_type.lxd.container
+    Scenario Outline: Run collect-logs on an attached machine
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        When I attach `contract_token` with sudo
+        And I run `python3 /usr/lib/ubuntu-advantage/timer.py` with sudo
+        And I verify that running `ua collect-logs` `as non-root` exits `1`
+        Then I will see the following on stderr:
+             """
+             This command must be run as root (try using sudo).
+             """
+        When I run `ua collect-logs` with sudo
+        Then I verify that files exist matching `ua_logs.tar.gz`
+        When I run `tar zxf ua_logs.tar.gz` as non-root
+        Then I verify that files exist matching `logs/`
+        When I run `ls -1 logs/` as non-root
+        Then stdout matches regexp:
+        """
+        cloud-id.txt
+        dmesg.txt
+        jobs-status.json
+        journalctl.txt
+        livepatch-status.txt
+        systemd-timers.txt
+        ua-auto-attach.path.txt
+        ua-auto-attach.service.txt
+        ua-reboot-cmds.service.txt
+        ua-status.json
+        ua-timer.service.txt
+        ua-timer.timer.txt
+        ubuntu-advantage.log
```

Comment:
Why are the `ubuntu-advantage-timer.log` not included ?

Reply:
aaaah I started on another branch and without the logging PR it wasn't there yet. Will rebase and include it.

Reply:
Added also the license-check log, expecting the GCP separate timer PR